Week1/homework/app.js

```diff
@@ -1,11 +1,51 @@
 'use strict';
 {
   const bookTitles = [
-    // Replace with your own book titles
-    'harry_potter_chamber_secrets'
+    { 'title': 'Becoming', 'language': 'English', 'author': 'Michelle Obama' },
+    { 'title': 'Those Who Knew', 'language': 'English', 'author': 'Scott Belsky' },
+    { 'title': 'Good and Mad', 'language': 'English', 'author': 'Rebecca Traister' },
+    { 'title': 'The Messy Middle', 'language': 'English', 'author': 'Scott Belsky' },
+    { 'title': 'Lake Success', 'language': 'English', 'author': 'Gary Shteyngart' },
+    { 'title': 'Fashion Climbing', 'language': 'English', 'author': 'Bill Cunningham' },
+    { 'title': 'Crudo', 'language': 'English', 'author': 'Olivia Laing' },
+    { 'title': 'Winners Take All', 'language': 'English', 'author': 'Anand Giridharadas' },
+    { 'title': 'French Exit', 'language': 'English', 'author': 'Patrick deWitt' },
+    { 'title': 'How To Be Famous', 'language': 'English', 'author': 'Caitlin Moran' }
   ];
 
+  function makeList() {
+    let b1 = document.createElement('list');
+    document.body.appendChild(b1);
 
-  // Replace with your own code
-  console.log(bookTitles);
-}
+    let ul = document.createElement('ul');
+    b1.appendChild(ul);
+
+    for (let i = 0; i < bookTitles.length; i++) {
+
+      let li = document.createElement('li');
+      li.setAttribute("id", "book" + [i]);
```

Comment:
I think you meant `"book"+i` (without the [ ]). The construct `[i]` creates an Array, which, luckily, when concatenated with a string does output the intended "book0", "book1" and so on, however, it is not needed